annotators/emotional_bot_response/server.py

```diff
@@ -0,0 +1,191 @@
+    moods, and intensity levels while keeping the original meaning and vocabulary range.
+    Feel free to adjust sentence structures and choose words that effectively convey
+    the desired emotions, moods, and intensity. Keep the language complexity simple.
+    Don't mention intensity level in the output. Preserve the type of the sentence
+    if it is a question.
+
+    **Output:**"""
+    return prompt
+
+
+def get_llm_emotional_response(prompt):
+    sending_variables = compose_sending_variables(
+        {},
+        ENVVARS_TO_SEND,
+    )
+    try:
+        hypotheses = send_request_to_prompted_generative_service(
+            "",
+            prompt,
+            GENERATIVE_SERVICE_URL,
+            GENERATIVE_SERVICE_CONFIG,
+            GENERATIVE_TIMEOUT,
+            sending_variables,
+        )
+        response = hypotheses[0]
+    except Exception as e:
+        sentry_sdk.capture_exception(e)
+        logger.exception(e)
+        response = None
+
+    return response
+
+
+def rewrite_sentences(sentence, bot_emotion, bot_mood_label):
+    result = {}
+    try:
+        prompt = make_prompt(sentence, bot_emotion, bot_mood_label, 7)  # emotion, mood, intensity
+        response = get_llm_emotional_response(prompt)
+        result = {"hypotheses": response}
+    except Exception as exc:
+        logger.exception(exc)
+        sentry_sdk.capture_exception(exc)
+        result = {"hypotheses": "fallback standard response"}
+    return result
+
+
+@app.route("/respond", methods=["POST"])
+def respond():
+    st_time = time.time()
+
+    sentences = request.json.get("sentences", [])
+    bot_mood_label = request.json.get("bot_mood_label", [])
+    bot_emotion = request.json.get("bot_emotion", [])
+
+    results = []
+    for i in range(len(sentences)):
+        result = rewrite_sentences(sentences[i], bot_emotion[0], bot_mood_label[0])
+        results.append(result)
+
+    total_time = time.time() - st_time
+    logger.info(f"emotional-bot-response exec time: {total_time:.3f}s")
+
+    return jsonify(results)
+
+
+@app.route("/respond_batch", methods=["POST"])
+def respond_batch():
+    st_time = time.time()
+
+    sentences = request.json.get("sentences", [])
+    bot_mood_label = request.json.get("bot_mood_label", [])
+    bot_emotion = request.json.get("bot_emotion", [])
+
+    results = []
+    for i in range(len(sentences)):
+        result = rewrite_sentences(sentences[i], bot_emotion[i], bot_mood_label[i])
+        results.append(result)
+
+    total_time = time.time() - st_time
+    logger.info(f"emotional-bot-response exec time: {total_time:.3f}s")
+
+    return jsonify([{"batch": results}])
+
+
+try:
+    logger.info("emotional-bot-response is starting")
+
+    sentences = ["I will eat pizza."]
+    bot_mood_labels = ["angry"]
+    bot_emotions = ["anger"]
+    responses = rewrite_sentences(sentences, bot_mood_labels, bot_emotions)
+    logger.info("TEST. Sentences: {}".format(sentences))
```

Comment:
more common to use f-string for now which is:
```
f"TEST. Sentences: {sentences}"
```

Reply:
fixed